app/decorators/decidim/admin/selective_newsletter_form_decorator.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+Decidim::Admin::SelectiveNewsletterForm.class_eval do
+  def map_model(_newsletter)
+    self.participatory_space_types = Decidim.participatory_space_manifests.map do |manifest|
+      Decidim::Admin::SelectiveNewsletterParticipatorySpaceTypeForm.from_model(manifest: manifest)
+    end.unshift(additional_participatory_space_manifest)
+  end
```

Comment:
```suggestion
self.participatory_space_types.unshift(additional_participatory_space_manifest)
```

Reply:
Like this? Rubocop complains about:  Redundant `self` detected.
```ruby
  def map_model(_newsletter)
    self.participatory_space_types = Decidim.participatory_space_manifests.map do |manifest|
      Decidim::Admin::SelectiveNewsletterParticipatorySpaceTypeForm.from_model(manifest: manifest)
    end
    participatory_space_types.unshift(additional_participatory_space_manifest)
  end
```

Reply:
yes, I think it's much more readable and easy to understand this way

Reply:
Thanks. Done.